dist/index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
```

Comment:
dist folder should be part of .gitignore,

delete the folder, git add dist/ and then add it into gitignore then do commit push